packages/app/src/cli/models/app/app.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
```

Comment:
I don't love blanketing files especially when we're trying to prevent new occurrences. I know the file is large too, can we get GPT to write us up a codemod/script to add individually? Same with a few other files

Reply:
Generally I tried to exercise some judgement with these -- if there were lots of dotted around breakages of this rule I did a file ignore rather than have loads of comments. I figure this was better than sacrificing readability too much. I don't really mind switching to line ignores really. Ultimately its in the eye of the beholder: Your call?

Another option would be to make these warnings. We don't use that right now, everything is errors or off. The downside of warnings is in net-new code, they can blow right through them if they like. Ignoring errors leaves a footprint. We'd have to get warnings to show up here in GH for those to be viable, I think